test/js/specs/happy.js

```diff
@@ -409,4 +409,158 @@ describe('Happy Paths', options, ({driver, pageObjects}) => {
       })
     })
   })
-})
\ No newline at end of file
+  describe('PROOF OF ADDRESS', async () => {
+
+    const goToPoADocumentSelectionScreen = async () => {
+      driver.get(localhostUrl + `?poa=true&async=false&useWebcam=false`)
+      welcome.primaryBtn.click()
+      poaIntro.clickStartVerificationButton()
+    }
+
+    const uploadFileAndClickConfirmButton = async (fileName) => {
+      documentUpload.getUploadInput()
+      documentUpload.upload(fileName)
+      documentUploadConfirmation.confirmBtn.click()
+    }
+
+    it('should verify UI elements of PoA Intro screen', async () => {
+      const poaIntroCopy = poaIntro.copy()
+      driver.get(localhostUrl + `?poa=true`)
+      welcome.primaryBtn.click()
+      poaIntro.verifyTitle('Let’s verify your UK address')
+      poaIntro.verifyRequirementsHeader(poaIntroCopy)
+      poaIntro.verifyFirstRequirement('Shows your current address')
+      poaIntro.verifySecondRequirement('Matches the address you used on signup')
+      poaIntro.verifyThirdRequirement('Is your most recent document')
+      poaIntro.verifyStartVerificationButton(poaIntroCopy)
+    })
+
+    it('should verify UI elements of PoA Document Selection screen', async () => {
+      const poaDocumentSelectionCopy = poaDocumentSelection.copy()
+      goToPoADocumentSelectionScreen()
+      poaDocumentSelection.verifyTitle('Select a UK document')
+      poaDocumentSelection.verifySubtitle(poaDocumentSelectionCopy)
+      poaDocumentSelection.verifyElementsBankCell(poaDocumentSelectionCopy)
+      poaDocumentSelection.verifyElementsUtilityBillCell(poaDocumentSelectionCopy)
+      poaDocumentSelection.verifyElementsCouncilTaxLetter(poaDocumentSelectionCopy)
+      poaDocumentSelection.verifyElementsBenefitsLetter(poaDocumentSelectionCopy)
+    })
+
+    it('should verify UI elements of PoA Guidance for Bank Statement', async () => {
+      const poaGuidanceCopy = poaGuidance.copy()
+      goToPoADocumentSelectionScreen()
+      poaDocumentSelection.clickOnBankIcon()
+      poaGuidance.verifyCopiesOnPoADocumentsGuidanceScreen(poaGuidanceCopy, 0)
```

Comment:
Passing an index is not very readable. I don't think the array should be in `pageobjects/PoaGuidance.js`. You can put the array in this file, iterate over it and pass the name as an argument.

Reply:
Applied changes in e82cbfd6647ef0a72cd67c794f67183d4ec83291 using correct doc keys directly instead of array.